lesson_13/maps_java/maps_app/src/main/java/com/codedifferently/lesson13/Lesson13.java

```diff
@@ -1,12 +1,25 @@
 package com.codedifferently.lesson13;
 
+import java.util.HashMap;
+
 public class Lesson13 {
 
-  /**
-   * Provide the solution to LeetCode 3146 here:
-   * https://leetcode.com/problems/permutation-difference-between-two-strings
-   */
   public int findPermutationDifference(String s, String t) {
-    return 0;
+    HashMap<Character, Integer> stringS = new HashMap<>();
+    HashMap<Character, Integer> stringT = new HashMap<>();
+    int diffSum = 0;
+    for (int i = 0; i < t.length(); i++) {
+      stringS.put(s.charAt(i), i);
+      stringT.put(t.charAt(i), i);
+    }
+    for (Character key : stringS.keySet()) {
+      if (stringT.containsKey(key)) {
```

Comment:
Don't think you need this line.

Reply:
Is it line 16 that you are referring to?